src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/judgeapproval/ApproveOrderServiceTest.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.judgeapproval;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval.JudgeApproval;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.DraftOrdersWrapper;
+
+import java.util.stream.Stream;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval.JudgeDecision.READY_TO_BE_SEALED;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval.JudgeDecision.REVIEW_LATER;
+
+@ExtendWith(MockitoExtension.class)
+class ApproveOrderServiceTest {
+
+    private static final CaseDocument TARGET_DOC = mock(CaseDocument.class);
+
+    @Spy
```

Comment:
No need for @Spy here

Reply:
[52ed03c](https://github.com/hmcts/finrem-case-orchestration-service/pull/1997/commits/52ed03c441dd1de348f5b56f899b7bfa46177be2)